Kevin-Liu-01.md

```diff
@@ -0,0 +1,78 @@
+components to create the circuits that will power the prosthetic hand. This will entail soldering sensors, motors, and other components to the breadboard.
+
+4. Assemble the hand: I will use the 3D printed parts and the assembled circuits to build the prosthetic hand. I will attach the motors and sensors to the hand, 
+and connect all circuits to my Arduino Nano.
+
+5. Testing: Once the hand is assembled, I will test it to make sure all the parts are working correctly and that the hand is able to perform the desired tasks. 
+This may involve adjusting the sensors and motors or writing some dummy code for the Arduino.
+
+6. Write the code: I will write some C++ loops for the Arduino that will control the prosthetic hand. The code will interpret signals from the sensors and use 
+that information to control the motors and make the hand move.
+
+7. Fine-tune and refine: As I test and use the prosthetic hand, I will most likely need to make adjustments and refinements to the design and code to make it more 
+effective and reliable. I am prepared to iterate and improve the hand as I go.
+
+Originally, I considered outsourcing the resin printing to a 3rd party company, but as I began brainstorming I realized that this would be too inefficient for my project.
+I would have little to no control over the printing process and would not be able to make changes and adjustments as needed. This is particularly unfortunate for 
+prototyping and testing multiple designs or making quick changes on the fly. I can only begin designing and testing a new iteration when the company is ready,
+forcing me to wait for the new design to arrive any time I make any design change, which is contrary to rapid prototyping and would most likely go over the 10 days
+I would have for this project. On top of that, I would have to pay for the shipping, (potentially) the resin, and a whole bunch of other costs, like service fees, 
+and I would not be able to present these costs upfront (a stipulation for a grant).
+
+As such, because it was in my  combined budget, I decided to buy a 3D printer and resin instead. It may take some time out of the 10 days to assemble, 
+but it will save me a lot of time in the prototyping and testing cycle. It will ultimately save me an extremely valuable amount of time in the development process 
+and allow me to quickly prototype and test different hand designs. 
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Pack of wires   | https://www.adafruit.com/product/1957 | $1.95  |
+|Soldering Iron Kit|https://www.amazon.com/Soldering-Kit-Temperature-Desoldering-Electronics/dp/B07GTGGLXN/ref=sr_1_6?crid=28QD0NK9LQJCB&keywords=soldering+iron&qid=1672283598&s=industrial&sprefix=soldering+iron%2Cindustrial%2C92&sr=1-6| $18.99|
+|Myoware Muscle Sensors	|https://www.amazon.com/gp/product/B01GU6QMLI/ref=ox_sc_act_title_1?smid=A1ZW8W51C7XKQ5&psc=1|$24.99|
```

Comment:
This is a pack of 50 MyoWare Electrodes. Would you also need some MyoWare reference cables and/or a MyoWare Muscle Sensor to connect them to the Arduino?

Reply:
Yeah sorry, I should've mentioned it but I have the required muscle sensors (https://www.sparkfun.com/products/retired/13723) from school. I only need the electrodes because I need to constantly swap them out for testing. I hope that helps!